test/sly-package-fu-tests.el

```diff
@@ -0,0 +1,54 @@
+;; -*- lexical-binding: t; -*-
+(require 'sly-package-fu "contrib/sly-package-fu")
+(require 'sly-tests "lib/sly-tests")
+
+(def-sly-test sly-package-fu-sly-import (initial-defpackage final-defpackage symbol)
+              "Check if importing `import` on `initial-defpackage` results in `final-defpackage."
```

Comment:
indentation's off here.  Indent this when `def-sly-test` is properly evaluated in your Emacs.

Reply:
Fixed indentation